_includes/configuration/contextmenu_never_use_native.md

```diff
@@ -1,12 +1,12 @@
 ## `contextmenu_never_use_native`
 
-The `contextmenu_never_use_native` option allows you to disable the browser's native context menu from appearing within the editor.
+The `contextmenu_never_use_native` option allows you to prevent the browser context menu from appearing within the editor.
 
-> Caution: Using this option may result into unexpected behavior when right-clicking in text areas. We advise you to consider all your options carefully before using this feature.
+> **Caution**: Using this option may result into unexpected behavior when right-clicking in text areas. We advise you to consider all your options carefully before using this feature.
```

Comment:
`may result into unexpected`? something is wrong here

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1600/commits/661608ce08831e40b62b55c36d8e4047b5b710e8